src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$card-margin-right: 48px;
+$border-color: #f3f3f3;
+$blue-accent-color: #00acdc;
+$dark-text-color: #060b35;
+$dark-grey-text-color: #616070;
+$white-color: #fff;
```

Comment:
Good practice is to name color variable in this convenction: **$color-colorName**, change variable names